extensions/spring-cloud-config-client/runtime/src/main/java/io/quarkus/spring/cloud/config/client/runtime/SpringCloudConfigClientConfig.java

```diff
@@ -129,6 +129,35 @@ public interface SpringCloudConfigClientConfig {
     @WithDefault("450")
     int ordinal();
 
+    /**
+     * Configuration for Config Server discovery.
+     */
+    DiscoveryConfig discovery();
+
+    interface DiscoveryConfig {
```

Comment:
I believe we should have another layer under `DiscoveryConfig` named `EurekaConfig`. The idea here being that in the future we might have some other way to do discovery (Stork for example).